Test/task1.c

```diff
@@ -0,0 +1,84 @@
+#include <stdio.h>
+#include <stdbool.h>
+
+unsigned int collection[1000];
+
+unsigned int fibonacci(int number);
+bool fibonacciTest();
+
+int main()
+{
+    if(!fibonacciTest())
+    {
+        return 1;
+    }
+    
+    int sum = 0;
+    
+    for(int i = 0; i < 100; i++)
+    {
+        unsigned int number = fibonacci(i);
```

Comment:
```suggestion
        const unsigned int number = fibonacci(i);
```
Это не ошибка, но с const было бы лучше. Многие стайлгайды (но не наш) требуют, чтобы всё, что может быть const, было const